lib/internal/util.js

```diff
@@ -431,6 +433,39 @@ function createDeferredPromise() {
   return { promise, resolve, reject };
 }
 
+let internalTimers;
+
+function getActiveResources() {
+  if (internalTimers == null) {
+    internalTimers = require('internal/timers');
+  }
+  const handles = process._getActiveHandles();
+  const reqs = process._getActiveRequests();
+
+  const timers = {};
+  ArrayPrototypeForEach(ObjectValues(internalTimers.timerListMap), (list) => {
+    var timer = list._idlePrev === list ? null : list._idlePrev;
```

Comment:
Is there a reason to use `var` here instead of `let`?

Reply:
I kept it because the original commit had it + I found this comment sprinkled over a number of places: https://github.com/nodejs/node/blob/88d9268d087eea0d5cd6e7b731b3679c266fb74f/lib/internal/async_hooks.js#L228-L229
I'm not sure whether it's a loop only optimisation though, is it?

Reply:
Well you're not using `for` loop anymore, I don't think it still applies. Also this is no a very hot path, is it?

Reply:
I don't think this is a hot path either. I'll add your suggestion.